arviz/rcparams.py

```diff
@@ -107,8 +169,10 @@ def _validate_boolean(value):
     "plot.bokeh.figure.width": (500, _validate_positive_int),
     "plot.bokeh.figure.height": (500, _validate_positive_int),
     "plot.bokeh.show": (True, _validate_boolean),
+    "plot.bokeh.marker": ("Cross", _validate_marker("bokeh")),
     "plot.matplotlib.constrained_layout": (True, _validate_boolean),
     "plot.matplotlib.show": (False, _validate_boolean),
+    "plot.matplotlib.marker": ("+", _validate_marker("matplotlib")),
```

Comment:
I would not include matplotlib marker here. matplotlib already has its own rcparams and marker can be handled there. Also, matplotlib markers allow even more options than these strings. Sticking with bokeh should be enough

Reply:
Okay, makes sense.